src/MassTransit/Serialization/JsonConverters/InterfaceJsonConverter.cs

```diff
@@ -0,0 +1,22 @@
+namespace MassTransit.Serialization.JsonConverters
+{
+    using System;
+    using System.Text.Json;
+    using System.Text.Json.Serialization;
+
+
+    public class InterfaceJsonConverter<TType, TImplementation> :
```

Comment:
hmm, does it duplicate TypeMappingJsonConverter logic ?

Reply:
No, it doesn't.